packages/web/src/components/gcds-footer/gcds-footer.tsx

```diff
@@ -66,29 +61,51 @@ export class GcdsFooter {
     return !!this.el.querySelector('[slot="list"]');
   }
 
+  private get hasContextual() {
+    return !!this.el.querySelector('[slot="contextual"]');
+  }
+
   render() {
-    const { lang, display, topHref, hasList, renderSignature } = this;
+    const { lang, display, hasList, hasContextual, renderSignature } = this;
     const govNav = I18N[lang].gov.menu;
+    const themeNav = I18N[lang].themes.menu;
     const siteNav = I18N[lang].site.menu;
 
     return (
       <Host role="contentinfo">
         <div class="top__container">
           <slot name="top"></slot>
         </div>
-
+        {hasContextual && 
+          <div class="gcds-footer__contextual">
+            <div class="contextual__container">
+              <slot name="contextual"></slot>
+            </div>
+          </div>}
         {display === "full" ?
           (<div class="gcds-footer__landscape">
-            <nav class="landscape__container" aria-label={I18N[lang].gov.heading}>
-              <h2>{I18N[lang].gov.heading}</h2>
-              <ul>
-                {Object.keys(govNav).map((value) =>
-                  <li>
-                    <a href={govNav[value].link}>{govNav[value].text}</a>
-                  </li>
-                )}
-              </ul>
-            </nav>
+            <div class="landscape__container">
+              <nav class="landscape__govnav" aria-label={I18N[lang].gov.heading}>
+                <h3>{I18N[lang].gov.heading}</h3>
+                <ul>
+                  {Object.keys(govNav).map((value) =>
+                    <li>
+                      <a href={govNav[value].link}>{govNav[value].text}</a>
+                    </li>
+                  )}
+                </ul>
+              </nav>
+              <nav class="landscape__themenav" aria-label={I18N[lang].themes.heading}>
+                <h4>{I18N[lang].themes.heading}</h4>
+                <ul>
+                  {Object.keys(themeNav).map((value) =>
+                    <li>
+                      <a href={themeNav[value].link}>{themeNav[value].text}</a>
+                    </li>
+                  )}
+                </ul>
+              </nav>
+            </div>
           </div>)
         : null }
 
```

Comment:
Looks like the footer brand is also using an `<h3>` now instead of the `<h2>`. They also updated the footer classes. The brand footer is now "gc-sub-footer" and the landscape footer is now "gc-main-footer".